src/resources/packages/classy/functions/utils.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Validates a URL string.
+ *
+ * @since TBD
+ *
+ * @param url - The URL string to validate.
+ * @returns True if the URL is valid, false otherwise.
+ */
+export function isValidUrl( url: string ): boolean {
+	if ( ! url || url.trim() === '' ) {
+		return true; // Allow empty URLs
+	}
+
+	try {
+		// Add protocol if missing for validation
```

Comment:
Here and elsewhere, add full-stops at the end of line comments.

Reply:
About comments, please trim them removing the ones explaining what the code is doing. Comment code if the "what" and "why" you're doing it is not obvious.